rts/Lua/LuaUnsyncedCtrl.cpp

```diff
@@ -4214,6 +4215,17 @@ int LuaUnsyncedCtrl::GarbageCollectCtrl(lua_State* L) {
 }
 
 
+/*** @function Spring.SetAutoShowMetal
+ * @bool autoShow
+ * @treturn nil
+ */
+int LuaUnsyncedCtrl::SetAutoShowMetal(lua_State* L)
+{
+	guihandler->autoShowMetal = !!luaL_checkboolean(L, 1);
```

Comment:
The cast isn't needed since both sides are bools.
```suggestion
	guihandler->autoShowMetal = luaL_checkboolean(L, 1);
```